src/test/java/seedu/address/logic/parser/CountryNoteAddCommandParserTest.java

```diff
@@ -23,12 +23,11 @@
     private final CountryNoteAddCommandParser parser = new CountryNoteAddCommandParser(tagNoteMap);
 
     @Test
-    public void parse_noCountryNoNote_throwsParseException() {
+    public void parse() {
```

Comment:
If its not too much work, I think it would be better to revert to the previous format of multiple test functions actually, since most of our test files use the multiple function method, and that's what the coding standards seem to recommend also.

Reply:
hmm true. i actually wanted to change all to EP style but its gonna take a lot of time